net/third_party/quiche/overrides/quiche_platform_impl/quiche_iovec_impl.h

```diff
@@ -11,10 +11,12 @@
 
 #if BUILDFLAG(IS_WIN) || defined(COMPILER_MSVC)
 /* Structure for scatter/gather I/O.  */
+
```

Comment:
nit: this seems like commit noise, please don't modify upstream unless we have to